index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="index.js" defer></script>
+    <style>            
+    @media screen and (max-width: 992px) {
+        body {
+        background-color: rgb(192, 192, 211);
+        }
+    }
+    </style>
+    <title>Document</title>
+</head>
+<body>
+    
+    <main id="main-doc" >
+        <nav id="navbar">
+            <header>Technical Documentation for programming lanaguages</header>
+            <a href="#javascript_and_java" class='nav-link' target="#javascript_and_java" >javascript and java</a>
+            <a href="#golang_and_coffee-script" class='nav-link' target="#golang_and_coffee-script">golang and coffee-script</a>
+            <a href="#typescript_and_c#" class='nav-link' target="#typescript_and_c#" >typescript and c#</a>
+            <a href="#css_and_html" class='nav-link' target="#css_and_html" >css and html</a>
+            <a href="#less_and_sass" class='nav-link' target="#less_and_sass" >less and sass</a>
+        </nav>
+        <section class="main-section" id="javascript_and_java" >
+            <header>javascript and java</header>
+            <code>Game</code>
+            <p></p>
+            <p></p>
+            <p></p>
+            <li></li>
```

Comment:
When `<li>` appears in the wild with no `<ul>`, `<ol>` or `<menu>` container, it causes HTML structural problems. Please wrap the other `<li>`'s in the page with a `<ul>` tag.